doc/io.rst

```diff
@@ -338,6 +338,36 @@ supported by netCDF4-python: 'standard', 'gregorian', 'proleptic_gregorian' 'nol
 By default, xarray uses the 'proleptic_gregorian' calendar and units of the smallest time
 difference between values, with a reference time of the first time value.
 
+.. _io.iris:
+
+Iris
+----
+
+The Iris_ tool allows easy reading of common meteorological and climate model formats
+(including GRIB and UK MetOffice PP files) into ``Cube`` objects which are in many ways very
+similar to ``DataArray``s, while enforcing a CF-compliant data model. If iris is
```

Comment:
Needs to be "`DataArray` object"

(RST can't handle anything adjacent to backticks which is why the color looks weird here in GitHub's syntax highlighting. Yes, it's not very intuitive.)

Reply:
Good catch, thanks